Rubberduck.Parsing/VBA/SelectedDeclarationProvider.cs

```diff
@@ -93,6 +99,21 @@ public Declaration SelectedDeclaration(QualifiedSelection qualifiedSelection)
             return SelectedModule(qualifiedSelection);
         }
 
+        private static ParameterDeclaration SelectedDeclarationViaArgument(QualifiedSelection qualifiedSelection, DeclarationFinder finder)
+        {
+            var members = finder.Members(qualifiedSelection.QualifiedName).Where(m => m.DeclarationType.HasFlag(DeclarationType.Procedure));
```

Comment:
It looks like selected arguments from Function and Properties are being excluded.  If that is not the intent, wouldn't
 `DeclarationType.HasFlag(DeclarationType.Member)` be correct here?

Reply:
At that stage in the procedure the idea is to identify the scope we're in, so excluding parameters here, is intentional.

Reply:
Perhaps I'm misunderstanding...Yes, parameters are excluded - but using `DeclarationType.Procedure` only includes Subroutines and Property Let/Set scopes. No?